internal/vulnstore/postgres/querybuilder_test.go

```diff
@@ -149,6 +165,13 @@ func Test_GetQueryBuilder_Deterministic_Args(t *testing.T) {
 			},
 		},
 	}
+
+	// This is safe to do because SQL doesn't care about what whitespace is
+	// where.
+	//
+	// Also, it produces more intelligible diffs when things break.
+	normalizeWhitespace := cmpopts.AcyclicTransformer("normalizeWhitespace", strings.Fields)
```

Comment:
clever!!

Reply:
Yeah, pretty happy with this one. Makes problems the tests _much_ more readable.